### ISSUE TITLE
Restrictive dietary pattern, excessive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10564]
**Preferred term label:**
Restrictive dietary pattern, excessive

**Synonyms**
Extremely restrictive diets often noted in children who refuse varied food types that can result in nutritional deficiencies; avoidant or restrictive food intake; extreme diet

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Various vitamin deficiencies

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It might be good to specify what the restriction is?

**Comment by MickeySegal** [comment]
This is widely recognized as "picky" eaters, and formalized in UMLS with https://uts.nlm.nih.gov/uts/umls/concept/C2228523 "excessive diet restriction".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001072
- label: Excessive diet restriction
- parents: [Nutrition history (HP:4000123)](https://hpo.jax.org/app/browse/term/HP:4000123)
- pmid: PMID:33171966
- def: A tendency to reject a variety of foods and/or to eat a restricted range of foods.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10564
- synonyms:
